paloesg/excide
# Remap roles in conductor

# Description

- What i did for remap users of role `contractor_in_charge`, `contractor`, `event_owner`, `manpower_allocator` using rake task, so you need to run `rake users:remap_roles_in_conductor`
- For change the Activation to Event, i create migration for change table name Activation to Event 
- Rename value of column `trackable_type` from `Activation` to `Event` in the table activities

i did change scheduler rake task from `contractor_reminders` to be `associate_reminder`, so maybe you want to update the scheduler

Trello link:
- https://trello.com/c/bHc1z0QD
- https://trello.com/c/zhB5bxFn

## Remarks

Filter needs to be updated.

# Testing

- run `rails db:migrate`
- run `rake users:remap_roles_in_conductor`
- run `rake update:activation_to_event`
- Check all function still working in conductor

## Checklist:
